application/android/app/src/main/java/com/group12/beabee/views/goals/GoalsPrototypeAdapter1.java

```diff
@@ -0,0 +1,118 @@
+package com.group12.beabee.views.goals;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.group12.beabee.R;
+import com.group12.beabee.models.responses.GoalShort;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class GoalsPrototypeAdapter1 extends RecyclerView.Adapter<GoalsPrototypeAdapter1.ViewHolder> {
+
+    private List<GoalShort> goalShortList;
+    private IOnGoalClickedListener onItemClickedListener;
+
+    public void setData(List<GoalShort> goalShorts){
+        goalShortList = goalShorts;
+        notifyDataSetChanged();
+    }
+
+    public void setItemClickListener(IOnGoalClickedListener listener){
+        onItemClickedListener = listener;
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+
+        if(viewType==0){
+            View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_goal_list_marketplace, parent, false);
+            return new ViewHolder(view);
+        }
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_goal_list_marketplace, parent, false);
```

Comment:
You probably dont need the second viewType and second view holder for this adapter